README.md

```diff
@@ -71,6 +71,23 @@ brew install cmdg
 ```
 
 ## Configuring
+You need to configure `cmdg` in order to provide it with authentication
+so it can talk to GMail on your behalf. To do this you'll need to generate
+a ClientID and ClientSecret. You can do this with the following steps:
+
+  1. Go to the [Google Developers Console](https://console.developers.google.com/apis).
+  1. Select an existing project or create a new project
+  1. Navigate to the "Credentials" tab.
+  1. Navigate to the "OAuth consent screen".
+  1. Fill out the OAuth consent screen.
+  1. Make sure to add scopes for the various APIs you'll need
+    1. Drive API - .../auth/drive.appdata
+    1. Gmail API - https://mail.google.com
```

Comment:
Scopes look right as enumerated, at least. I'm surprised the gmail one isn't `../auth/gmail.modify`, but it may have changed since I set up my key.

Matches the [scopes in the source code](https://github.com/ThomasHabets/cmdg/blob/master/pkg/cmdg/connection.go#L33)

Reply:
It could be that ../auth/gmail.modify works. I went for the broadest scope. I'm going to try toning down to ../auth/gmail.modify and see if I encounter any problems.